.azure-pipelines/Linux-CI.yml

```diff
@@ -42,10 +42,8 @@ jobs:
 
   - script: |
       python -m pip install --upgrade pip
-      python -m pip install virtualenv
-      python -m virtualenv py$(python.version)
-      source py$(python.version)/bin/activate
       python -m pip install numpy protobuf
+      python -m pip numpy
```

Comment:
I don't understand the solution

Reply:
I think the goal would be make mypy only check onnx's directory. ONNX should not take care of its dependencies' typecheck... I removed virtualenv because if using it, the installed numpy will be produced in the current dircetory and mypy will check that as well. 

However, even if we remove it, mypy will still catch the type error from numpy in other path... Still investigation.